<?xml version="1.0"?>

<launch>
	
  <arg name="config_path" default="${cobot_gui}/config"/>
  <arg name="file_name" default="default.conf"/> 

  <arg name="emergency_stop_topic_name" default="/emergency_stop/state"/> 
  <arg name="diagnostics_toplevel_topic_name" default="/diagnostics_toplevel_state"/> 
  <arg name="diagnostics_topic_name" default="/diagnostics"/> 
 
  <node pkg="cobot_gui" type="cobot_gui_node.py" name="cobot_gui_node" output="screen">
    <param name="config" value="$(arg config_path)/$(arg file_name)"/>
  </node>
  
</launch>
